Allow ingesting from a non-default S3 endpoint (#3084)
treeverse/lakeFS: docs/setup/import.md

The `lakectl ingest` command will attempt to use the current user's existing credentials and will respect instance profiles,
environment variables and credential files [in the same way that the AWS cli does](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html){: target="_blank" }

> If you wish to ingest from other S3 compatible storage solution you could use the flag to specify the endpoint `--s3-endpoint-url https://play.min.io`

Reviewer: I would shorten phrasing here:
```suggestion
Specify an endpoint to ingest from other S3 compatible storage solutions, e.g. add `--s3-endpoint-url https://play.min.io`.
```
